### ISSUE TITLE
Cifar-10 acc  value is decreasing or fixed in keras model

### ISSUE THREAD

**Issue by ErmiaAzarkhalili** [fchollet/keras#4669]
Please make sure that the boxes below are checked before you submit your issue. Thank you!

- [x] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps

- [x] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [x] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

@farizrahman4u @fchollet I saw the issue once before, but there was no explanation there (https://github.com/fchollet/keras/issues/1478). you only mentioned that it would be better to utilize your own model in example folder...

when you create keras model (https://github.com/fchollet/keras/blob/master/examples/cifar10_cnn.py), and increase the batch_size from 32 to 128, the acc when running the example is decreasing or fixed after a while regradless of the number of convolution layers you used...

could you please explain what's the issue here?

**Comment by fchollet** [comment]
I've changed the default optimizer in this script to RMSprop (instead of SGD). Update your Keras version from Github.

I believe it's possible that some weight initialization configurations could randomly cause training not start when using SGD with this specific learning rate.

**Comment by olivierhub** [comment]
I have the same issue,
when running cifar10_cnn.py, at epoch 6 loss is increasing.

Any hint about what could cause this?

tne script used is exactly: 
https://github.com/fchollet/keras/commit/309f586424ea229fa9f6fa963e9345b4dbd22849

```
pip list | egrep -i "kera|theano"
Keras (1.2.1)
Theano (0.9.0.dev5)
```

```
python -ucifar10_cnn.py
Using Theano backend.
Using gpu device 0: GeForce GTX 960 (CNMeM is enabled with initial size: 3000 MB, cuDNN 5105)
X_train shape: (50000, 3, 32, 32)
50000 train samples
10000 test samples
Using real-time data augmentation.
Epoch 1/200
50000/50000 [==============================] - 27s - loss: 1.6221 - acc: 0.4121 - val_loss: 1.1710 - val_acc: 0.5854
Epoch 2/200
50000/50000 [==============================] - 27s - loss: 1.2532 - acc: 0.5541 - val_loss: 1.0700 - val_acc: 0.6173
Epoch 3/200
50000/50000 [==============================] - 27s - loss: 1.1437 - acc: 0.5995 - val_loss: 1.1454 - val_acc: 0.6073
Epoch 4/200
50000/50000 [==============================] - 27s - loss: 1.1094 - acc: 0.6189 - val_loss: 0.9723 - val_acc: 0.6580
Epoch 5/200
50000/50000 [==============================] - 27s - loss: 1.1066 - acc: 0.6235 - val_loss: 0.9107 - val_acc: 0.6837
Epoch 6/200
50000/50000 [==============================] - 27s - loss: 1.1163 - acc: 0.6262 - val_loss: 0.8780 - val_acc: 0.7001
Epoch 7/200
50000/50000 [==============================] - 27s - loss: 1.1605 - acc: 0.6157 - val_loss: 0.9086 - val_acc: 0.6882
Epoch 8/200
50000/50000 [==============================] - 27s - loss: 1.2074 - acc: 0.6042 - val_loss: 1.0281 - val_acc: 0.6460
Epoch 9/200
50000/50000 [==============================] - 27s - loss: 1.2579 - acc: 0.5883 - val_loss: 1.0236 - val_acc: 0.6471
Epoch 10/200
50000/50000 [==============================] - 28s - loss: 1.3126 - acc: 0.5749 - val_loss: 1.1547 - val_acc: 0.6283

```

**Comment by al3xsh** [comment]
@fchollet @olivierhub I have exactly the same issue as this using the latest version of cifar10_cnn.py  (commit 309f586).

As this issue is closed, I have reported it as #5239.

Regards,

Alex
